### DISCUSSION TITLE
v3.2.125

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#14901] [General]
## What's Changed
* Fix typo in 02-stats-overview.md by @jackwh in https://github.com/filamentphp/filament/pull/14790
* [Docs] Correct minor typo in widgets 02-stats-overview.md by @drbyte in https://github.com/filamentphp/filament/pull/14789
* Add missing malay translation by @sumardi in https://github.com/filamentphp/filament/pull/14796
* Add missing section "Lifecycle hooks" for ViewRecord by @abrardev99 in https://github.com/filamentphp/filament/pull/14795
* Improve table header cell design by @zepfietje in https://github.com/filamentphp/filament/pull/14797
* Added missing fields of Turkish language by @apicopter in https://github.com/filamentphp/filament/pull/14798
* Added missing file for Turkish language by @apicopter in https://github.com/filamentphp/filament/pull/14799
* Fix typo in icons docs by @nhedger in https://github.com/filamentphp/filament/pull/14829
* Add nestedRecursiveRules to ToggleButtons by @tanthammar in https://github.com/filamentphp/filament/pull/14815
* Avoid division by zero in FileUpload.php by @tanthammar in https://github.com/filamentphp/filament/pull/14831
* Add missing Indonesian translations by @uthadehikaru in https://github.com/filamentphp/filament/pull/14834
* Bump cross-spawn from 7.0.3 to 7.0.6 in /docs-assets/app by @dependabot in https://github.com/filamentphp/filament/pull/14859
* Fix toggle buttons CSS class by @zepfietje in https://github.com/filamentphp/filament/pull/14862
* Add automatic scrolling of wizard steps by @austincarpenter in https://github.com/filamentphp/filament/pull/14826
* Update indicator.blade.php - format $allSelectableRecordsCount by @ericmp33 in https://github.com/filamentphp/filament/pull/14838
* Improve sync connection support for `ExportAction` by @ralphjsmit in https://github.com/filamentphp/filament/pull/14854
* Add input and affix CSS classes to input wrapper by @zepfietje in https://github.com/filamentphp/filament/pull/14874
* Simplify color picker preview HTML by @zepfietje in https://github.com/filamentphp/filament/pull/14877
* Add CSS class to rich editor editor by @zepfietje in https://github.com/filamentphp/filament/pull/14879
* Add CSS class to tags input by @zepfietje in https://github.com/filamentphp/filament/pull/14880
* Add missing Slovak translations by @christeyerl in https://github.com/filamentphp/filament/pull/14881
* Add missed UK translation by @dmitry-udod in https://github.com/filamentphp/filament/pull/14899
* Make getLocaleLabelUsing usable in service provider by @johanmolen in https://github.com/filamentphp/filament/pull/14898
* Use Heroicons for rich editor and Markdown editor by @zepfietje in https://github.com/filamentphp/filament/pull/14878
* Update tabs/item.blade.php, add justify-center by @avinmaster in https://github.com/filamentphp/filament/pull/14888
* Lock Livewire to [v3.5.12](https://github.com/livewire/livewire/releases/tag/v3.5.12) due to reported issues with tables in v3.5.13.

## New Contributors
* @abrardev99 made their first contribution in https://github.com/filamentphp/filament/pull/14795
* @apicopter made their first contribution in https://github.com/filamentphp/filament/pull/14798
* @nhedger made their first contribution in https://github.com/filamentphp/filament/pull/14829
* @uthadehikaru made their first contribution in https://github.com/filamentphp/filament/pull/14834
* @johanmolen made their first contribution in https://github.com/filamentphp/filament/pull/14898
* @avinmaster made their first contribution in https://github.com/filamentphp/filament/pull/14888

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.124...v3.2.125

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.125'>v3.2.125</a>.</em>

**Comment by angus-mcritchie** [comment]
What are the reported issues with Livewire 3.5.13?

Looking to update Livewire in my project and I can't as filament is locked.

Happy to look at writing a fix if you can share the bug report? 🙂

**Reply by ralphjsmit** [reply]
Hello! It was reported in eg https://discord.com/channels/883083792112300104/956270111176679516/1311365505776488458 and https://discord.com/channels/883083792112300104/1313209203564871710/1313209203564871710.

**Reply by jackwh** [reply]
Since 3.5.13 there's been two PRs merged and released which might have fixed the issue already:

1. https://github.com/alpinejs/alpine/pull/4450
2. https://github.com/livewire/livewire/pull/9072

Can anyone confirm? If so, hopefully we can revert the version lock 🤞

**Reply by angus-mcritchie** [reply]
@jackwh I just installed Filament 3.2.124 and Livewire 3.5.17 and the black screen on a table action (sort, pagination etc.) is still happening.

After a bit of digging, I think it's related to this [Alpine JS commit](https://github.com/alpinejs/alpine/pull/4428/commits/463f2770ed5efb66b6487c9a33dfb7c8d116765d) which doesn't init components inside a `x-ignore` attribute.

I have made a [pull request](https://github.com/filamentphp/filament/pull/15044). If anyone else can test it, that would be great.

**Reply by danharrin** [reply]
We aren't able to do that unfortunately, so we will need another fix or Alpine needs to revert the breaking change.

**Reply by angus-mcritchie** [reply]
Thanks for looking into this @danharrin. Big fan of your work!

I just raised [this question](https://github.com/alpinejs/alpine/pull/4428#issuecomment-2531493568) in the Alpine JS repo. 🤞 we can get this working again.

**Reply by jackwh** [reply]
Thanks @angus-mcritchie, I'd recommend posting it as an issue too so it doesn't get missed. Hope it's resolved soon 🤞

**Reply by danharrin** [reply]
https://github.com/filamentphp/filament/pull/15062

**Reply by danharrin** [reply]
Sorry guys, we have had to lock it again due to further issues reported across multiple components.

**Reply by danharrin** [reply]
Please see #15559

**Comment by tahinuralam** [comment]
Nice.....
